modules/solid_mechanics/include/materials/SolidModel.h

```diff
@@ -199,6 +203,9 @@ class SolidModel : public DerivativeMaterialInterface<Material>
   // Compute quantity used in thermal term of J Integral
   virtual void computeThermalJvec();
 
+  //Compute thermal expansion, used in thermal term of interaction integral
+  virtual void computeThermalExpansion();
```

Comment:
I also think this should be re-named. Maybe `computeCurrentInstantaneousThermalExpansionCoefficient`?